src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -83,6 +93,11 @@ private static void OnProcessExit(object sender, EventArgs e)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
+            if (_logger.IsDefaultPlatformLoggingEnabled)
```

Comment:
Don't use `IsDefaultPlatformLoggingEnabled`. We should probably get rid of it, it's poorly documented. 

Use `_logger.IsLoggingEnabled()`

Reply:
CC @trwalke for help here.

Reply:
Quite possibly we should always assume a logger exists...

Reply:
Spoke with @trwalke .Please check the changes made. We can discuss if you disagree.